editor/import/resource_importer_scene.cpp

```diff
@@ -1537,6 +1725,7 @@ void ResourceImporterScene::get_import_options(const String &p_path, List<Import
 		script_ext_hint += "*." + E;
 	}
 
+	r_options->push_back(ImportOption(PropertyInfo(Variant::FLOAT, "nodes/scene_scale", PROPERTY_HINT_RANGE, "0.001,1000,0.001"), 1.0));
```

Comment:
Why introduce a new scale option? Why not a toggle for "Bake root scale" ?

Reply:
This sort of discussion might be better suited for the proposal itself. I think the idea was to remove or deprecate the old scene scale which would make the checkbox redundant.

Reply:
Resolved because it should be in a separate pr.